rizza/helpers/genetics.py

```diff
@@ -34,6 +34,7 @@ def _breed_pair(self, gene_list1, gene_list2):
         Random mutation is then given a chance to change the child.
 
         """
+        # TODO now with nested lists and dict this requires rework
```

Comment:
this is now the bottleneck which requires additional work. As mutation in some cases does not work the program will fail, but it shouldn't be a lot of work to fix it.